graphql/generator/object.go

```diff
@@ -770,6 +773,8 @@ func genFieldResolverTypeCoercion(t ast.Type, i info, nullable bool) jen.Code {
 		return jen.Qual(defsPkg, "Int").Op(".").Id("ParseValue").Call(jen.Id("val")).Assert(jen.Int())
 	case graphql.Float.Name():
 		return jen.Qual(defsPkg, "Float").Op(".").Id("ParseValue").Call(jen.Id("val")).Assert(jen.Float64())
+	case graphql.ID.Name():
+		fallthrough
```

Comment:
I think this is redundant, since nothing is changed before the fallthrough.

Reply:
👍 replaced